test/e2e/cluster_extension_install_test.go

```diff
@@ -75,6 +75,7 @@ func TestClusterExtensionInstallRegistry(t *testing.T) {
 	clusterExtension.Spec = ocv1alpha1.ClusterExtensionSpec{
 		PackageName:      "prometheus",
 		InstallNamespace: "default",
+		ServiceAccount:   "default",
```

Comment:
I have a suspicion that the default ServiceAccount won't have the permissions necessary to install an extension. We may need to ensure we are creating a new ServiceAccount and the appropriate RBAC resources to install the e2e extensions

Reply:
I just realized that this is irrelevant for this particular PR because it is just adding the new field and doesn't actually implement the logic 🤦. Apologies for this distraction - I don't know what I was doing and why I thought this would be the case...